sources/packages/load-test/src/_tests_/workflow/workflow-assessment-submission.test.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Methods like setup and default are part of the K6 life cycle.
+ * @see https://k6.io/docs/using-k6/test-lifecycle
+ * The exported options are part of the K6 configuration.
+ * @see https://k6.io/docs/using-k6/k6-options
+ */
+import { check, sleep } from "k6";
+import {
+  workflowAssessmentSubmission,
+  workflowLoadAssessmentData,
+} from "../../utils/load-test-api/load-test-api";
+import { Options } from "k6/options";
+import execution from "k6/execution";
+import { getLoadTestGatewayCredentials } from "../../utils/load-test-api/load-test-api-creds";
+
+/**
+ * Load test number of iterations to run.
+ */
+const ITERATIONS = 50;
+/**
+ * Virtual users to run load test.
+ * Please ensure that the number of virtual users
+ * must always be less than the iterations.
+ */
+const VIRTUAL_USERS = 15;
+
+interface SetupData {
+  assessmentIds: number[];
+}
+
+/**
+ * Load all the required data as per the required iterations
```

Comment:
I had a simular question on the API side, IMO would be clear to call it "Create" data instead "Load".